pkg/koordlet/statesinformer/states_noderesourcetopology.go

```diff
@@ -0,0 +1,176 @@
+/*
+Copyright 2022 The Koordinator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package statesinformer
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/k8stopologyawareschedwg/noderesourcetopology-api/pkg/apis/topology/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/util/retry"
+	"k8s.io/klog/v2"
+	"k8s.io/kubernetes/pkg/kubelet/cm/cpuset"
+
+	"github.com/koordinator-sh/koordinator/apis/extension"
+	"github.com/koordinator-sh/koordinator/pkg/koordlet/metriccache"
+)
+
+func (s *statesInformer) syncNodeResourceTopology(node *corev1.Node) {
+	topologyName := fmt.Sprintf("koord-%s", node.Name)
```

Comment:
It is better to re-using node name for NodeResourceTopology like NodeMetrics, NodeSLO, etc.

Reply:
In order to be compatible with the existing NodeResourceTopology instances and prevent koordlet from conflicting with existing components.

Reply:
In fact, we expect koordlet to be an alternative to topology gathering, and if that's the case, it's better to keep the name the same.

Reply:
> In fact, we expect koordlet to be an alternative to topology gathering, and if that's the case, it's better to keep the name the same.

OK。 LGTM.